synthtool/gcp/templates/java_library/.kokoro/nightly/integration.cfg

```diff
@@ -1,37 +1,31 @@
 # Format: //devtools/kokoro/config/proto/build.proto
 
-# Configure the docker image for kokoro-trampoline.
+{%- if 'defaults' in metadata and 'kokoro_nightly_env_vars' in metadata['defaults'] -%}
```

Comment:
It looks as if the logic only works for kokoro nightly builds environment variables. Does this meet Kristen's issue?

Reply:
I'll send a separate message.